go/vt/vtgate/planbuilder/operators/logical.go

```diff
@@ -268,16 +267,17 @@ func getOperatorFromJoinTableExpr(ctx *plancontext.PlanningContext, tableExpr *s
 }
 
 func getOperatorFromAliasedTableExpr(ctx *plancontext.PlanningContext, tableExpr *sqlparser.AliasedTableExpr) (ops.Operator, error) {
+	tableID := ctx.SemTable.TableSetFor(tableExpr)
 	switch tbl := tableExpr.Expr.(type) {
 	case sqlparser.TableName:
-		tableID := ctx.SemTable.TableSetFor(tableExpr)
+		tableID := tableID
```

Comment:
`tableID := tableID` this can probably be removed.

Reply:
![image](https://user-images.githubusercontent.com/402248/233994361-a1ed022b-7a28-4c23-9efb-b03aeca5ba8a.png)